.github/workflows/release-debug-base.yaml

```diff
@@ -0,0 +1,38 @@
+name: Create Debug Base Image Release
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
```

Comment:
So I think there's a slight race here. If I push a change that touches apko.yaml and some Go code, the workflow to build the base image and the workflow to build the image on top of it will start separately at the same time. The base image workflow hopefully finishes first, but there's no guarantee. The result would be an inconsistently based crane debug image, which might be confusing and hard to track down.

Can this apko step just happen directly in the workflow to rebuild the crane debug image? It doesn't take long, and it should produce the same digest if packages haven't changed, so I don't mind doing it even if apko.yaml hasn't changed.

Actually that's another reason -- if the `busybox` package has changed in alpine but the apko.yaml hasn't, I'd still like to rebuild the image, so let's run this step as often as we can to try to pick up those package updates.

Reply:
totally agree with you, I made some changes according to your feedbacks, thanks for bringing them to my attention.